pkg/identity/email/principal.go

```diff
@@ -40,6 +42,11 @@ func PrincipalFromIDToken(ctx context.Context, token *oidc.IDToken) (identity.Pr
 		return nil, errors.New("email_verified claim was false")
 	}
 
+	emailRegex := regexp.MustCompile(`^.+@.+\..+$`)
```

Comment:
Can we use something like https://github.com/asaskevich/govalidator/blob/f21760c49a8d602d863493de796926d2a5c1138d/validator.go#L67 instead? 

the regex they use is a bit more complicated (and potentially more thorough?) - https://github.com/asaskevich/govalidator/blob/f21760c49a8d602d863493de796926d2a5c1138d/patterns.go#L7

Reply:
if not, then can we move this to a const or done in `init()` so we aren't compiling the regex on every call?

Reply:
done